packages/wallet/contract/src/smartWallet.js

```diff
@@ -50,9 +50,13 @@ export const makeSmartWallet = async (
   const notifierParts = {
     contacts: E(admin).getContactsNotifier(),
     dapps: E(admin).getDappsNotifier(),
+    /** all current issuers */
     issuers: E(admin).getIssuersNotifier(),
+    /** recently updated offers */
```

Comment:
"recently" seems unnecessarily vague. It's exactly the latest offer in some cases and it's all offers when `updateAllState` is called (whenever petnames are added or changed).